### PR TITLE
Show progress as int

### PR DESCRIPTION
**Pull request by jjnicola**

### CODE REVIEW

File: ospd/ospd.py [hunk 1]
```diff
@@ -602,20 +602,26 @@ def sort_host_finished(
...
 
         scan_progress = self.calculate_progress(scan_id)
         self.scan_collection.set_progress(scan_id, scan_progress)
 
     def set_scan_host_progress(
-        self, scan_id: str, host: str = None, progress: int = None,
+        self,
+        scan_id: str,
+        host: str = None,
+        progress: Union[float, int] = None,
```
Comment (bjoernricks) [inline ospd/ospd.py:616]: Why should be a float still allowed?

File: ospd/scan.py [hunk 2]
```diff
@@ -274,7 +275,12 @@ def get_count_alive(self, scan_id: str) -> int:
     def get_current_target_progress(self, scan_id: str) -> Dict[str, int]:
         """ Get a scan's current dead host count. """
 
-        return self.scans_table[scan_id]['target_progress']
+        hosts = self.scans_table[scan_id]['target_progress']
+        if hosts:
+            for host, progress in hosts.items():
+                hosts[host] = int(progress)
```
Comment (bjoernricks) [inline ospd/scan.py:281]: Shouldn't we ensure that progress is already an int in the scans table?

File: ospd/ospd.py (hunk 1 above)
Comment (jjnicola) [reply ospd/ospd.py:616]: This comes from the wrapper. It depends on the wrapper implementation. Currently ospd-openvas sends a float value

Comment (bjoernricks) [reply ospd/ospd.py:616]: Can we fix that?

**Pull request merged**